SpeechRecognizerButton/SFButton.swift

```diff
@@ -43,7 +43,13 @@ open class SFButton: UIButton {
                                                       AVNumberOfChannelsKey: 1,
                                                       AVEncoderAudioQualityKey: AVAudioQuality.high.rawValue]
     @IBInspectable public var maxDuration: Double = 60
-    public var locale = Locale.autoupdatingCurrent
+    public var locale = Locale.autoupdatingCurrent {
+        didSet {
```

Comment:
Maybe `willSet { speechRecognizer = nil }` is easier to understand and have the same result?